app/src/main/java/com/yeonae/chamelezone/view/home/HomeActivity.kt

```diff
@@ -16,7 +16,7 @@ import com.yeonae.chamelezone.view.mypage.MypageTabFragment
 import kotlinx.android.synthetic.main.activity_home.*
 
 class HomeActivity : AppCompatActivity() {
-    private val tabList by lazy { listOf("홈", "코스", "지도", "즐겨찾기", "MY") }
+    private val tabList by lazy { listOf("홈", "코스", "지도", "좋아요", "MY") }
```

Comment:
```suggestion
    private val tabList by lazy { resource.getStringArray(R.array.main_tab) }
```
string-array resource를 사용해보는 것도 좋을 것 같습니다.

Reply:
수정했습니다